### ISSUE TITLE
Checksum emoji is not shown in short form

### ISSUE THREAD

**Issue by stringhandler** [tari-project/wallet-ios#1111]
Build v0.27.0 b746

In the address screen there is a checksum emoji, but it is not shown in the short form of the address. This checksum is specifically there to help visually identify your address, and should be shown

**Comment by TruszczynskiA** [comment]
No, to identify the address you need a network and spend key because these two address parts doesn't change over time.
